sink_service/source/main.c

```diff
@@ -60,17 +60,109 @@ static bool get_service_name(char service_name[MAX_SIZE_SERVICE_NAME], unsigned
     return true;
 }
 
+/**
+ * \brief   Attempts to acquire an environment variable
+ * \param   dest
+ *          The destination buffer where to store the read value
+ * \param   name
+ *          The name of the environment variable to acquire
+ * \return  A pointer to the character buffer
+ */
+static bool get_env_var(char * dest, char * name)
+{
+    char * ptr = NULL;
+    int ret;
+
+    memset(dest, '\0', sizeof(*dest) * MAX_SIZE_PARAMETER);
+    if ((ptr = getenv(name)) != NULL)
+    {
+        ret = snprintf(dest, sizeof(*dest) * (MAX_SIZE_PARAMETER - 1), "%s", ptr);
+        if (ret < 0)
+        {
+            LOGW("Input var (%s) too long for destination\n", name);
+            return false;
+        }
+        LOGI("%s: %s\n", name, dest);
```

Comment:
Maybe message could be more explicit. Something like: Environment variable read %s: %s